keda/2.7.1/manifests/cma.v2.7.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,590 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Consider adding to the commit message the command used to generate the  modified files. 
For example:
```
Ran: 
  cma-generate-csv.sh 2.7.1

```

Reply:
That's a good idea. Will do!